crowdsource/resolvers/person.py

```diff
@@ -0,0 +1,91 @@
+from opencivicdata.core.models import Person
+from dataquality.models import DataQualityIssue, IssueResolverPatch
+
+
+# apply approved patches and create new issues, in case a scraper scrapes a new value
+# which doesn't match with old_value of patch(which can be ignored later if scraping was correct)
+def apply_person_patches(jur_name):
+    count = 0
+    patches = IssueResolverPatch.objects.filter(jurisdiction__name=jur_name,
+                                                status='approved')
+    image_duplicates = []
+    name_duplicates = []
+    for patch in patches:
+        person = Person.objects.get(id=patch.object_id)
+        issue = None
+        if patch.category == 'image':
+            # Number of approved pathces for 'image' for a person must be one.
+            ap = patches.filter(object_id=person.id, category='image').count()
```

Comment:
variable name should be more descriptive

Reply:
agreed.